work.ipynb

```
@@ -0,0 +1,699 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "bfb68478",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch\n",
+    "from torch_geometric.datasets import TUDataset\n",
+    "from torch_geometric.utils import to_networkx\n",
+    "import networkx as nx\n",
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/AIDS"

(S108)